extramodules/it/eng/jhove/module/png/PngModule.java

```diff
@@ -155,6 +155,15 @@
     // Standard keyword for the creation timestamp
     private final static String CREATION_TIME_KEYWORD = "Creation Time";
 
+    private final static int MAX_INT = 0x7FFFFFFF;
+
+    // Color Type
+	private final static int EachPixel_GRAYSCALE = 0;
+	private final static int EachPixel_RGB = 2;
+	private final static int EachPixel_PALETTA = 3;
```

Comment:
Small typo perhaps? EachPixel_PALETTE might be better, sorry.

Reply:
yep :) the problem when I programming at midnight